Makefile

```diff
@@ -314,13 +314,11 @@ test: test-unit ##@tests Run basic, short tests during development
 test-unit: export BUILD_TAGS ?=
 # Ensure 'waku' and 'wakuv2' tests are executed first to reduce the impact of flaky tests.
 # Otherwise, the entire target might fail at the end, making re-runs time-consuming.
-test-unit: export UNIT_TEST_PACKAGES ?= $(shell go list ./... | grep -E '/waku(/.*|$$)|/wakuv2(/.*|$$)') \
```

Comment:
@cammellos should we re-enable wakuv2 tests?, line 324 `grep -E -v '/wakuv2(/.*|$$)')` is effectively skipping them